src/bracket-ipv6-address.cpp

```diff
@@ -0,0 +1,70 @@
+// @file bracket-ipv6-address.c
+//
+// This tool takes an IP address as a string.  If the IP address is IPv4 it
```

Comment:
Normally I think we put these sort of introductory comments after the copyright statement (on the principle that the copyright statement should come first and be unmissable to anyone wanting to use this?).